### DISCUSSION TITLE
URL parameters change will not trigger query

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#4059] [Ideas]
Hello,

I use URL to pass parameters to my query, and I have some predefined URL filter like `View Active Customer` then it will run `Navigate To` a predefined URL like `/customers/list/CustomerID=10000&Status=Active`, it suppose will refresh the page then my query will parse the URL parameters and refresh data provider. But it seems that the URL parameters change will not trigger any page reload, and there's no way to use javascript to force a page reload, the users has to manually refresh the page which is not convenient. 

Is it possible to let the Budibase refresh if the URL made any change? Or add a button click action to force refresh page?

**Comment by mjashanks** [comment]
Hi! This is a bug, which I have just logged: https://github.com/Budibase/budibase/issues/4071

A workaround is to use a `Link` component. The drawback for that is that you cannot use any other actions as you navigate (i.e. `Save Row`, `Execute Query` etc)
